### PR TITLE
Notify Integration

### PR DESCRIPTION
**Pull request by alice-carr**
#### What problem does the pull request solve?
End User needs to be able to press a button that sends an email using Notify, including the title, text input, and submission time. To do this we need to integrate Notify into the runner. 

This PR is a work in progress

relates #21 

#### Checklist

- [x] I've used the pull request template
- [x] I've linked this PR to the relevant issue (if mission work)

### CODE REVIEW

**Comment by alice-carr** [comment]
I've created a template to use for this and added this into the PR.
The linting is currently failing because the method has too many lines - I could fix this by removing the check for the api key but that doesn't feel like the best thing to do. 

I would also appreciate if someone could tell me if `forms-runner/app/services/` is the right place for this file, or is there somewhere better for it?

**Review by danielburnley — APPROVED** [review]
Seems good to me for now - I can imagine we'll probably revisit some of this a lil when we start the actual work on the runner but as part of this spike it works well 🎉

File: .rubocop.yml [hunk 1]
```diff
@@ -34,4 +34,12 @@ Style/Documentation:
...
 Style/FrozenStringLiteralComment:
+  Enabled: false
+
+# Analog of: 736b3d295f88b9ba6676fc168b823535582388c2
+# "Disable opinionated cops"
+#
+# We should avoid cops that are based on heuristics, since
+# it's not clear what action to take to fix an issue.
+Metrics/MethodLength:
   Enabled: false
```
Comment (0atman) [inline .rubocop.yml:45]: I wonder if we should standardise on a fairly long method length? I agree that rubocop's defaults are very restrictive, but I'd really hate to scroll through many pages of methods, right?

**Review by 0atman — APPROVED** [review]
Looks good :ok_hand:

File: .rubocop.yml (hunk 1 above)
Comment (0atman) [reply .rubocop.yml:45]: Certainly shouldn't block this pr :ok_hand:

Comment (0atman) [reply .rubocop.yml:45]: captured https://github.com/alphagov/forms/issues/27

**Pull request merged**